### DISCUSSION TITLE
Option to switch between MiB and MB

### DISCUSSION THREAD

**Discussion by fabianmakila** [pelican-dev/panel#325] [Ideas]
Originally was going to suggest switching back to the old ways of displaying used storage as MB seeing how MiB has caused a lot confusion since it's introduction. However some people might actually prefer the MiB approach.

So my suggestion would be to add a switch to the admin panel (or a config file) that defines in which format to show these.

**Comment by lancepioch** [comment]
This has been made into a pr!
